pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,66 @@
+{ mkDerivation
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  qp = "./src/utils/qpaeq";
+in
+mkDerivation rec {
+  pname = "pulseuadio-qpaeq-${version}";
+  version = "13.0";
+
+  /* src = fetchurl {
+    url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
+    sha256 = "0mw0ybrqj7hvf8lqs5gjzip464hfnixw453lr0mqzlng3b5266wn";
+  }; */
+
+  src = ./.;
+
+  buildInputs = [
+    (
+      (python3.withPackages(ps: with ps; [ pyqt5 dbus-python sip ]))
+      .override(args: { ignoreCollisions = true; })
+    )
+  ];
+
+  nativeBuildInputs = [
+    pulseaudio.out
+  ];
+
+  installPhase = ''
+    install -D ${qp} $out/bin/qpaeq
+    '';
+
+  postPatch = ''
+    mkdir -p "$(dirname "${qp}")"
+    cp ${pulseaudio.out}/bin/qpaeq ${qp}
+
+    sed 's|dbus.mainloop.pyqt5|dbus.mainloop.glib|g' -i ${qp}
+    sed 's|DBusQtMainLoop|DBusGMainLoop|g' -i ${qp}
+    '';
+
+  preFixup = ''
+    wrapQtApp $out/bin/qpaeq
+    '';
+
+  meta = {
+    description = "Sound server for POSIX and Win32 systems";
```

Comment:
This is not correct, am I right? Because this package is only qpaeq...

Same goes for the `longDescription`.

Reply:
This package was more of a "I have no clue which way to go forward with that, let's just make a PR to see if splitting qpaeq vs adding it to pulse via flag is a good idea"

Therefore it's pretty broken